transcom/mymove
# B-21525 - Update email: ppm has been processed

## From Agility Ticket [B-21525](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21525)

## Testing Steps I took
1. Start the local server with the following command:
```
EMAIL_BACKEND=ses AWS_REGION=us-gov-west-1  aws-vault exec transcom-gov-dev -- make server_run
```
2. Start the local client.
3. As Service Member (Army), create a new PPM move and send to Service Counsellor.
Note: Make sure you use an email address that you have access to and have a checkmark in the "Email" checkbox for preferred contact method.
4. As Service Counsellor, update the orders and approve the move.
5. You should receive an email to your email account that reads (spacing may be slightly different but should still look appropriate. The tags that start with '<<' and end with '>>' should have those logical values in stead of the actual text shown here):
```
*** DO NOT REPLY directly to this email ***
This is a confirmation that your Personally Procured Move (PPM) with the assigned move code <<THE MOVE LOCATOR>> from <<THE ORIGIN CITY>>, <<THE ORIGIN STATE>> to <<THE DESTINATION CITY>>, <<THE DESTINATION STATE>> has been processed in MilMove.

Next steps:

For Army personnel (FURTHER ACTION REQUIRED):
You can now log into MilMove https://my.move.mil/ and download your payment packet to submit to the Defense Finance and Accounting Service (DFAS). You must complete this step to receive final settlement of your PPM.
Note: Not all claimed expenses may have been accepted during PPM closeout if they did not meet the definition of a valid expense.

Please be advised, your local finance office may require a DD Form 1351-2 to process payment. You can obtain a copy of this form by utilizing the search feature at https://www.esd.whs.mil.
If you have any questions, contact a government transportation office. You can see a listing of transportation offices on Military One Source here: https://installations.militaryonesource.mil/search?program-service=2/view-by=ALL

Thank you,
USTRANSCOM MilMove Team
The information contained in this email may contain Privacy Act information and is therefore protected under the
Privacy Act of 1974.  Failure to protect Privacy Act information could result in a $5,000 fine.
```
